common/commands/config.go

```diff
@@ -143,7 +150,26 @@ func (cc *ConfigCommand) ServerDetails() (*config.ServerDetails, error) {
 }
 
 func (cc *ConfigCommand) CommandName() string {
-	return "config"
+	isOidcConfigured, err := clientUtils.GetBoolEnvValue(coreutils.UsageOidcConfigured, false)
+	if err == nil && isOidcConfigured {
+		return ConfigOidcCommandName
+	}
+	return ConfigCommandName
+}
+
+// ExecAndReportUsage runs the ConfigCommand and then triggers a usage report if needed,
+// Report usage only if OIDC integration was used
+// Usage must be sent after command execution as we need the server details to be set.
+func (cc *ConfigCommand) ExecAndReportUsage() error {
```

Comment:
To simply things, I suggest we always report the usage here for the `jf config` command, regardless if OIDC is used or not. Of curse, we're using two different command names for OIDC and non-OIDC config.